File: C/2020/12限/array3.c
//  array3.c
//  cl  array3.c
#include <stdio.h>

int main() {
    int i[3][4] = {
        {1,2,3,4},
        {5,6,7,8},
        //{12,34,56,78},
    };
    //
    for(int j = 0; j < 3; j++) {
        for(int k = 0; k < 4; k++) {
            printf("%d ", i[j][k]);
        }
        printf("\n");
    }

    return 0;
}

File: GameServer/2026/Sat34/array3.php
<?php  // array3.php
echo "<pre>";

$hash_arr = [
    "str" => 10,
    "dex" => 15,
    "wiz" => 20,
];
print_r($hash_arr);

//
foreach ($hash_arr as $k => $v) {
    // $v *= 2; // これだとダメ
    $hash_arr[$k] = $v * 2;
}
print_r($hash_arr);

$arr = [
    0 => 0,
    1 => 11,
    2 => 222,
    3 => 3333,
];
print_r($arr);

$arr2 = [
    3 => 3,
    1 => 11,
    0 => 0,
    2 => 222,
    99 => 9999,
];
print_r($arr2);
